### PR TITLE
Renaming to Carbon.Security.

### PR DESCRIPTION
**Pull request by splatteredbits**

### CODE REVIEW

File: Carbon.Security/en-US/about_Carbon.Permissions.help.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    editions.
+
+LONG DESCRIPTION
+    SYSTEM REQUIREMENTS
+
+    * Windows PowerShell 5.1 and .NET 4.6.1+
+    * PowerShell 7+
+
+
+    INSTALLING
```
Comment (decoyjoe) [inline Carbon.Security/en-US/about_Carbon.Permissions.help.txt:15]: Is this meant to be indented?

File: Carbon.Security/en-US/about_Carbon.Permissions.help.txt [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        Install-Module -Name 'Carbon.Security'
+        Import-Module -Name 'Carbon.Security'
+
+    To install privately:
+
+        Save-Module -Name 'Carbon.Security' -Path '.'
+        Import-Module -Name '.\Carbon.Security'
+
+
+    COMMANDS
```
Comment (decoyjoe) [inline Carbon.Security/en-US/about_Carbon.Permissions.help.txt:28]: Is this meant to be indented?

**Review by decoyjoe — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -19,23 +18,17 @@ keys/key containers.
...
 
 ## Commands
 
 * `Get-CAcl`: Gets the access control (i.e. security descriptor) for a file, directory, or registry key.
-* `Get-CPermission`: Gets the permissions (access control rules) for a file, directory, registry key, or certificate
-  private key/key container.
-* `Grant-CPermission`: Grants permissions on a file, directory, registry key, or certificate private key/key container.
-* `Revoke-CPermission`: Revokes permissions on a file, directory, registry key, or certificate private key/key
-  container.
-* `Test-CPermission`: Tests permissions on a file, directory, registry key, or certificate private key/key container.
```
Comment (decoyjoe) [inline README.md:41]: Did you mean to remove these? These functions still exist in the module.

File: Carbon.Security/en-US/about_Carbon.Permissions.help.txt (hunk 2 above)
Comment (splatteredbits) [reply Carbon.Security/en-US/about_Carbon.Permissions.help.txt:28]: Yes. It's how PowerShell's `about` help topics are expected to be formatted.

Comment (splatteredbits) [reply Carbon.Security/en-US/about_Carbon.Permissions.help.txt:28]: I stand corrected. It's how PowerShell's about help topics *used* to be formatted.

File: README.md (hunk 3 above)
Comment (splatteredbits) [reply README.md:41]: Yes. But they are considered internal functions and shouldn't be used by regular folk, only by other Carbon modules. If people find them, fine, but they should be using `Carbon.FileSystem`, `Carbon.Registry`, and `Carbon.Cryptography` instead. The docs on each function explain that.

Comment (decoyjoe) [reply README.md:41]: Ah ok, good to know. I saw them exported in the `.psd1` and expected them to be public.

**Pull request merged**